scripts/selector-declaration-generator.js

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
+const fs = require("fs");
+const { readFromDir } = require("docworks-repo");
+const $wDeclarationBuilder = require("./selector-declaration-builder");
+
+async function generate$wdtsFile() {
+  const corvidDocsPath = "corvid_docs";
```

Comment:
consider making `corvid_docs` a paremeter

Reply:
why not use some unique temporary path for the cloned docs ?